ansible/group_vars/nfs

```diff
@@ -0,0 +1,3 @@
+nfs_fstype: "xfs"
```

Comment:
Just  a heads-up, this group name is going to change in the next stage, and when that change takes effect we'll want to update the name of this file to `nfs_server` to match.